common/djangoapps/util/log_sensitive.py

```diff
@@ -69,7 +69,16 @@ def encrypt_for_log(message, reader_public_key_b64):
     Returns a string <sender public key> "|" <ciphertext> wrapped in
     some framing text "[encrypted: ...]"; the inner string can be
     decrypted with decrypt_log_message.
+
+    For ease of use, key may be None or empty; a warning message will be
+    returned instead of data, encrypted or otherwise.
     """
+    # If no key was configured, don't raise an error. This method is
+    # expected to be called from debugging code, so it shouldn't blow
+    # up on misconfiguration.
+    if not reader_public_key_b64:
+        return '[encryption failed, no key]'
```

Comment:
Should we print the name of the key setting, or is that already done?

Reply:
This method only receives the value of the key, not the name. I did consider having the caller pass in the setting name instead, but that removes some flexibility.

Reply:
Makes sense. I like your choice. Someone will know, or they can track down from the rest of the message, so not a big deal.